objects.js

```diff
@@ -2779,7 +2829,7 @@ SpriteMorph.prototype.reporterize = function (expressionString) {
 
     if (expressionString > 100) {return null; }
```

Comment:
More for @jmoenig, but I also feel like these magic numbers should be given meaningful names. :)

Also I'm pretty sure this should be `expressionString.length`?